src/components/Comments/WriteComment.js

```diff
@@ -116,10 +133,12 @@ const WriteComment = ({
         <TouchableOpacity
           testID="iscommentenable"
           onPress={() => onPress(isAnonimity, anonimityData)}
-          style={styles.btn(isDisableSubmit || loadingUser)}
-          disabled={isDisableSubmit || loadingUser}>
+          style={styles.btn(!isCommentEnabled || loadingUser || loadingPost)}
+          disabled={!isCommentEnabled || loadingUser || loadingPost}>
           <MemoSendComment
-            fillBackground={isDisableSubmit || loadingUser ? '#C4C4C4' : colors.bondi_blue}
+            fillBackground={
+              !isCommentEnabled || loadingUser || loadingPost ? '#C4C4C4' : colors.bondi_blue
+            }
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The change in the condition for setting the style and disabling the button seems to be correct, as it now considers both `loadingUser` and `loadingPost`. However, please ensure that this change does not affect other parts of the application where this component is used.


<!-- This is an auto-generated comment by OSS CodeRabbit -->